app/scripts/services/error.js

```diff
@@ -6,6 +6,8 @@ angular.module('confRegistrationWebApp')
       // Generate a promise catch handler that generates an Error object from an HTTP response object
       errorFromResponse: function (defaultErrorMessage) {
         return function (res) {
+          console.error(res);
```

Comment:
Was this console on purpose? Just making sure it wasn't left over from development.